packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -203,14 +209,16 @@ export class Chip {
           </ic-typography>
           {dismissible && (
             <ic-tooltip
-              label="Dismiss"
+              label={dismissLabel ? dismissLabel : "Dismiss"}
```

Comment:
```suggestion
              label={dismissLabel}
```
I think this can just be dismissLabel as the default is 'Dismiss'